codegen-test/model/misc.smithy

```diff
@@ -199,3 +200,42 @@ structure ResponseCodeRequiredOutput {
     @httpResponseCode
     responseCode: Integer,
 }
+
+@idempotent
+@http(method: "GET", uri: "/test-accept-header")
+@documentation("Service accepts `*` in ACCEPT header")
+@httpRequestTests([
+    {
+        id: "AcceptHeaderStarRequestTest",
+        protocol: "aws.protocols#restJson1",
+        uri: "/test-accept-header",
+        headers: {
+            "Accept": "application/*",
+        },
+        params: {},
+        body: "{}",
+        method: "GET",
+        appliesTo: "server",
+    },
+    {
+        id: "AcceptHeaderStarStarRequestTest",
+        protocol: "aws.protocols#restJson1",
+        uri: "/test-accept-header",
+        headers: {
+            "Accept": "*/*",
+        },
+        params: {},
+        body: "{}",
+        method: "GET",
+        appliesTo: "server",
+    }
+])
+operation AcceptHeaderStarService {
+    input: AcceptHeaderStarServiceInput,
+    output: AcceptHeaderStarServiceOutput,
+}
+
+@output
+structure AcceptHeaderStarServiceOutput {}
+@input
+structure AcceptHeaderStarServiceInput {}
```

Comment:
I think we should dispense of superfluous modeling in the protocol tests, even if they are best practices, to keep the files minimal so readers can easily focus on the modeling aspects of the test that are strictly relevant to what it is testing.

So, e.g. not using `@input/@output`, using a single `AcceptHeaderStarServiceInputOutput` structure, not using `@idempotent` etc.